src/main/java/com/gmail/yuramitryahin/model/dto/OrderResponseDto.java

```diff
@@ -0,0 +1,19 @@
+package com.gmail.yuramitryahin.model.dto;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private LocalDateTime orderDate;
```

Comment:
```suggestion
    private String orderDate;
```

Reply:
Bohdan said that we better send LocalDateTime